content/docs/2.7/concepts/authentication.md

```diff
@@ -233,21 +237,28 @@ Key Vault [documentation](https://docs.microsoft.com/en-us/azure/key-vault/gener
 The `clientId` and `tenantId` for the application
 are to be provided as part of the spec. The `clientSecret` for the application is expected to be within a secret on the cluster.
 
+The `cloud` parameter can be used to specify cloud environments besides `Azure Public Cloud`, such as known Azure clouds like
+`Azure China Cloud`, etc. and even Azure Stack Hub or Air Gapped clouds.
+
 ```yaml
-azureKeyVault:                                      # Optional
-  vaultURI: {key-vault-address}                     # Required
-  credentials:                                      # Required
-    clientId: {azure-ad-client-id}                  # Required
-    clientSecret:                                   # Required
-      valueFrom:                                    # Required
-        secretKeyRef:                               # Required
-          name: {k8s-secret-with-azure-ad-secret}   # Required
-          key: {key-within-the-secret}              # Required
-    tenantId: {azure-ad-tenant-id}                  # Required
-  secrets:                                          # Required
-  - parameter: {param-name-used-for-auth}           # Required
-    name: {key-vault-secret-name}                   # Required
-    version: {key-vault-secret-version}             # Optional
+azureKeyVault:                                          # Optional.
+  vaultURI: {key-vault-address}                         # Required.
+  credentials:                                          # Required.
+    clientId: {azure-ad-client-id}                      # Required.
+    clientSecret:                                       # Required.
+      valueFrom:                                        # Required.
+        secretKeyRef:                                   # Required.
+          name: {k8s-secret-with-azure-ad-secret}       # Required.
+          key: {key-within-the-secret}                  # Required.
+    tenantId: {azure-ad-tenant-id}                      # Required.
+  cloud:                                                # Optional.
+    type: AzurePublicCloud | AzureUSGovernmentCloud | AzureChinaCloud | AzureGermanCloud | Private # Required.
+    keyVaultResourceURL: {kv-resource-url-for-cloud}    # Required when type = Private.
```

Comment:
Nit - Let's be consistent with the other placeholders and ensure people understand what it means instead of using abbreviations
```suggestion
    keyVaultResourceURL: {key-vault-resource-url-for-cloud}    # Required when type = Private.
```

Reply:
Done.